tests/e2e/docker/init-wp-beta.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "Initializing WooCommerce E2E"
+
+wp plugin install woocommerce --activate
+wp theme install twentynineteen --activate
+wp user create customer customer@woocommercecoree2etestsuite.com --user_pass=password --role=customer --path=/var/www/html
+
+# we cannot create API keys for the API, so we using basic auth, this plugin allows that.
+wp plugin install https://github.com/WP-API/Basic-Auth/archive/master.zip --activate
+
+echo "Updating to WordPress Nightly"
+
+wp plugin install wordpress-beta-tester --activate
```

Comment:
Just noting that by default `wordpress-beta-tester`seems to use the Nightly for the Point Release branch not trunk. We would need to use something like `wp core update --version=trunk` if we wanted bleeding edge Nightly. Not sure if that was the intention of this script so maybe we can clarify which Nightly branch we are trying to update to in the `echo`.

Reply:
@tammullen Thanks for catching that. `089bb87` clarifies that it's the point release.